docs/HowTo/Get-Started/System-Requirements.md

```diff
@@ -30,6 +30,24 @@ To determine system requirements, check CPU and disk space requirements using
 Besu requires 4GB of RAM. For public Ethereum networks, including MainNet and testnets such as
```

Comment:
You should either remove or modify this sentence that contradicts your new text
Also why do you have this change also in #276 ?

Reply:
Aren't RAM requirements and JVM memory requirements different things? Perhaps @EdJoJob you can help clarify?

Reply:
The RAM requirement should be the same as the JVM memory requirement, they are the same thing.  However people may interpret the RAM as what is available on the physical machine, not on the JVM.  We should be calling it JVM memory everywhere and the numbers should be 8G for public nets and 4G for private ones as a minimum

Reply:
Thanks for your help on this @EdJoJob. Have updated the JVM numbers and removed the RAM section.